plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -591,12 +591,16 @@ codemodernizer.chat.message.transform_cancelled_by_user=I cancelled your transfo
 codemodernizer.chat.message.transform_in_progress=If I run into any issues, I might pause the transformation to get input from you on how to proceed.
 codemodernizer.chat.message.transform_stopped_by_user=I stopped your transformation. If you want to start another transformation, choose **Start a new transformation**.
 codemodernizer.chat.message.transform_stopping=I'm stopping your transformation...
+codemodernizer.chat.message.upload_failed_connection_refused=Sorry, I couldn't upload your project to begin the transformation. Please check your network connectivity or firewall configuration, and then try again.
+codemodernizer.chat.message.upload_failed_http_error=I'm sorry, I couldn't upload your project to start the transformation. You can investigate the issue with the HTTP Status Code: {0}
```

Comment:
A while ago we had an issue where using apostrophe here led to the {0} not being replaced if I remember correctly. So as a precaution we decided to just not use apostrophes/contractions anywhere in our messages.

Reply:
Oh I see. Our tech writer asked to use apostrophe so I replaced the usage. I can see it being a potential issue if what goes in {0} contains apostrophe though. I'll change them back.